docs/CHANGELOG.md

```diff
@@ -29,6 +29,7 @@ The format is based on [Keep a Changelog][kc], and this project adheres to
 * Derive `PartialEq` for `MaterialPrimitive` ([#809])
 * Make `with_bindings_from_file` return a Result ([#811])
 * Logger initialization is now optional and can be enabled with a call to `amethyst::start_logger()` ([#815])
+* Gamepad support with optional builtin SDL controller event source ([#818])
```

Comment:
You also need to create the # 818  link. See how the other are doing it in the same file.

Reply:
Oh, forgot about that. Added